app/ui-react/syndesis/src/modules/support/pages/SupportPage.tsx

```diff
@@ -0,0 +1,36 @@
+import { WithApiVersion } from '@syndesis/api';
+import { PageSection, SimplePageHeader, SupportPageBody } from '@syndesis/ui';
+import * as React from 'react';
+import { PageTitle } from '../../../shared';
+
+export function SupportPage() {
```

Comment:
`export const SupportPage: React.FunctionComponent = () => {` would be more explicit and in line with the rest of the code.